user-guide/Advanced_Functionality/Databases/STaaS/STaaS_cost_estimation.md

```diff
@@ -6,9 +6,7 @@ uid: STaaS_cost_estimation
 
 To request a cost estimation, follow the procedure below:
 
-1. Follow the [setup procedure](xref:Setting_up_StaaS) until you come to the step where you need to wait to receive confirmation of your registration.
-
-1. Wait until you have received confirmation of your registration by email.
+1. Follow the [setup procedure](xref:Setting_up_StaaS) to register your system to use STaaS and stop after step 5.
```

Comment:
As the numbering of procedures can easily change, it's safer not to refer to the numbers of the steps.
```suggestion
1. Follow the [STaaS setup procedure](xref:Setting_up_StaaS), but only up to the point where your system is registered to use STaaS.
```